### PR TITLE
Migrate tokenlists to new config format

### PR DESCRIPTION
**Pull request by timjrobinson**
# Description

Move Tokenlists from the constants file to the new config files in directories. 

## Type of change

- [x] Code refactor / cleanup

## How should this be tested?

- Ensure that the app builds the tokenlists correctly, and loads tokens on all networks. 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/lib/config/mainnet/voting-gauge-pools.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const votingGaugePools: string[] = [
+  '0x06df3b2bbb68adc8b0e302443692037ed9f91b42000000000000000000000063',
+  '0x072f14b85add63488ddad88f855fda4a99d6ac9b000200000000000000000027',
+  '0x0b09dea16768f0799065c475be02919503cb2a3500020000000000000000001a',
+  '0x186084ff790c65088ba694df11758fae4943ee9e000200000000000000000013',
+];
+
+export default votingGaugePools;
```
Comment (garethfuller) [inline src/lib/config/mainnet/voting-gauge-pools.ts:8]: Why is this included here?

**Comment by timjrobinson** [comment]
Tested by generating tokenlists with develop vs this branch and the output files are exactly the same

**Review by garethfuller — APPROVED** [review]
